README.md

```diff
@@ -1,131 +1,14 @@
-Contribution Guides to the CNCF Ecosystem
-=========================================
+# CNCF Contributors Website
 
-Welcome! Are you interested in contributing to one of CNCF hosted projects? This repository should help you. It contains information and guidelines about contributions to CNCF projects.
+Welcome! Are you interested in contributing to one of CNCF hosted projects? This
+repository contains information and guidelines about contributions to CNCF
+projects.
 
-CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
+<p align="center"><a href="https://contribute.cncf.io/contributors/">Contribution Guides to the CNCF Ecosystem</a></p>
```

Comment:
This link points to https://github.com/cncf/contribute/graphs/contributors, we have to fix this a bit :)

Reply:
Oops, yeah this assumed that we could merge the PR and quickly have the domain serve this new site. Let me change this to allow for some delay between merging and go-live

Reply:
@carolynvs we've merged this, please fix this.